addons/im_livechat/tests/test_get_operator.py

```diff
@@ -8,8 +8,9 @@
 
 @odoo.tests.tagged("-at_install", "post_install")
 class TestGetOperator(HttpCase):
-    def _create_operator(self, lang_code=None, country_code=None):
+    def _create_operator(self, lang_code=None, country_code=None, expertise=None):
```

Comment:
It should at least be plural or have an _ids suffix. This would make sense when testing multiple expertise (e.g., one operator has expertise in two required areas, while another operator has expertise in both).

Reply:
We don't want to refer to recordsets with `_ids`, only variable with ids inside them.

Fields are exception, reasoning is because they are stored as list of ids in db or something like that.

Reply:
The non-plural was a typo, this is fixed now.